mlir/test/Conversion/SCFToOpenMP/scf-to-openmp.mlir

```diff
@@ -1,9 +1,9 @@
-// RUN: mlir-opt -convert-scf-to-openmp %s | FileCheck %s
+// RUN: mlir-opt -convert-scf-to-openmp='num-threads=4' %s | FileCheck %s
 
 // CHECK-LABEL: @parallel
 func.func @parallel(%arg0: index, %arg1: index, %arg2: index,
           %arg3: index, %arg4: index, %arg5: index) {
-  // CHECK: omp.parallel {
+  // CHECK: omp.parallel num_threads(%[[NTH:.*]] : i32) {
```

Comment:
Would be interesting, but I don't know how to do it since I would need to check that the value of the operand is equal to 4 and I don't know how can I do that with CHECK .